platform-service/pyproject.toml

```diff
@@ -15,6 +15,8 @@ dependencies = [
     "unstract-flags @ file:///${PROJECT_ROOT}/../unstract/flags",
     "requests>=2.31.0",
     "singleton-decorator~=1.0.0",
+    "unstract-sdk~=0.54.1",
```

Comment:
NIT: Consider specifying such unpublished changes as [VCS dependencies](https://pdm-project.org/en/latest/usage/dependency/#vcs-dependencies) if you wish to test easily

Reply:
hmmmm.... Thank you fo pointing to this. I am just wondering if this is already supported where we can pick up a version from a particular git repo's branch (hope that is right), then why do we need to publish anything to local pypi / editable version installs etc for testing? Guess my understanding is not fully right @chandrasekharan-zipstack ?

Reply:
@gaya3-zipstack yes, that was my point. We needn't publish to pypi for the sake of testing. Its okay to have release candidates though, no harm. Just wanted to share this option with you for your own testing / for co-ordinating better with others